### ISSUE TITLE
App icon not work 

### ISSUE THREAD

**Issue by MrOlolo** [carrotquest/android-sdk#10]
App icon not work when i use name `ic_launcher.png`:
_AndroidManifest.xml:_
 ```
<application
        ...
        android:icon="@mipmap/ic_launcher"
        ...

```
![image](https://user-images.githubusercontent.com/24739054/109060606-aa87f000-76f6-11eb-9c8d-e22217551e1f.png)

if i delete from `android/build.gradle` that dependency(below) everything works fine:
`implementation 'io.carrotquest:android-sdk:1.0.29-commonRelease'`

Also i can change icon name (e.g. `ic_launcher1`) and everything works fine.

Device: Galaxy s10e. Android 10
Sdk: 1.0.29-commonRelease ( also checked 1.0.1,  1.0.25, 1.0.28,)

**Comment by botalov** [comment]
Hi, we found a resource with a default name that was causing the conflict with your app's icon. The resource was deleted and won't be included in the next release. Please try to update as soon as the release is rolled out.

**Comment by botalov** [comment]
Hi! It's fixed in  v1.0.57
